Makefile

```diff
@@ -14,56 +14,69 @@ WORKER_SRC = $(shell find packer/worker -type f)
 
 # Outputs
 PLATFORM_BOX = travis-platform_virtualbox_$(VERSION).box
-WORKER_BOX = travis-worker_virtualbox_$(VERSION).box
-
+WORKER_PRECISE_BOX = travis-worker_precise_virtualbox_$(VERSION).box
+WORKER_TRUSTY_BOX = travis-worker_trusty_virtualbox_$(VERSION).box
 
 # ----- Run commands for both platform and worker
 
-all: 		platform 			worker
-install: 	platform.install 	worker.install
-test: 		platform.test 		worker.test
-release:	platform.release	worker.release
-clean:		platform.clean		worker.clean
+all: 		platform 			worker/precise			worker/trusty
+install: 	platform.install 	worker/precise.install	worker/trusty.install
+test: 		platform.test 		worker/precise.test		worker/trusty.test
+release:	platform.release	worker/precise.release	worker/trusty.release
+clean:		platform.clean		worker/precise.clean	worker/trusty.clean
 
 .PHONY: all install test release clean
 
 
 # ----- Run commands on platform or worker individually
 
 platform: $(PLATFORM_BOX)
-worker: $(WORKER_BOX)
-.PHONY: platform worker
+worker/precise: $(WORKER_PRECISE_BOX)
+worker/trusty: $(WORKER_TRUSTY_BOX)
+.PHONY: platform worker worker.precise worker.trusty
```

Comment:
Shouldn't this be `worker/precise` and `worker/trusty` ? Instead of the period-separated names.